src/Symfony/Component/AssetMapper/ImportMap/ImportMapConfigReader.php

```diff
@@ -57,6 +57,7 @@ public function getEntries(): ImportMapEntries
                 isDownloaded: isset($data['downloaded_to']),
                 type: $type,
                 isEntrypoint: $isEntry,
+                version: $data['version'] ?? null,
```

Comment:
`version` needs to be written also if it's not null in `writeEntries()`.

Reply:
sure. it's done.